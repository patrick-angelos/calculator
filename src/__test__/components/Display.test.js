import React from 'react';
import renderer from 'react-test-renderer';
import Display from '../../components/Display';
import { render } from '@testing-library/react';
import  '@testing-library/jest-dom';

describe('Display', () => {
  it('renders correctly', () => {
    const tree = renderer
    .create(<Display />)
    .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('renders with default value 0', () => {
    const { container } = render(<Display />);
    const display = container.querySelector('div');
    expect(display).toHaveTextContent('0');
  });
});