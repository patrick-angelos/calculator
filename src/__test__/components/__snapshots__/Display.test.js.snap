// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display renders correctly 1`] = `
<div
  className="display height-100"
>
  0
</div>
`;
